lib/matchers/matchers.rb

```diff
@@ -252,6 +256,8 @@ def try_match(actual, op, expected) # rubocop:disable Metrics/CyclomaticComplexi
       return actual.send(op, expected.to_i)
     elsif expected.is_a?(String) && boolean?(expected) && [true, false].include?(actual)
       return actual.send(op, to_boolean(expected))
+    elsif boolean?(expected) && %w[true false].include?(actual)
```

Comment:
Minor lint offense: literals should be delimited by { and }.
`%w[true false]` to `%w{true false}` would fix the offense.